railties/lib/rails/engine.rb

```diff
@@ -36,10 +36,9 @@ module Rails
   #
   # == Configuration
   #
-  # Besides the +Railtie+ configuration which is shared across the application, in a
-  # <tt>Rails::Engine</tt> you can access <tt>autoload_paths</tt>, <tt>eager_load_paths</tt>
-  # and <tt>autoload_once_paths</tt>, which, differently from a <tt>Railtie</tt>, are scoped to
-  # the current engine.
+  # The +Railtie+ configuration is shared across the application. With a <tt>Rails::Engine</tt>
+  # you can access <tt>autoload_paths</tt>, <tt>eager_load_paths</tt> and <tt>autoload_once_paths</tt>,
+  # which are scoped to the current engine.
```

Comment:
I added a full stop after `application.` 

Then started the next sentence, WDYT?

Reply:
The comma is necessary because "In addition to the configuration shared by all railties and the application" is not a complete sentence.

I added a suggestion for a different way to break it into two sentences.

Reply:
Your new suggestion LGTM, this needed two sentences to clarify the point 👌